containers/pom.xml

```diff
@@ -84,6 +75,20 @@
 
   <profiles>
     <profile>
+      <id>default</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
```

Comment:
There is no alternative to this. To make sure we do not cause any issue, we should limit the use for reactor order and modules only.